pkg/controller/controller.go

```diff
@@ -61,28 +62,32 @@ func New(cfg Config) *Controller {
 	}
 }
 
-func (c *Controller) handleClusterEvent(event *Event) error {
+func (c *Controller) handleClusterEvent(event *Event) (bool, error) {
```

Comment:
add a comment for the returned value?

Reply:
use named values instead (as you previously suggest :) )